libraries/AP_Motors/AP_MotorsHeli_RSC.h

```diff
@@ -147,6 +151,7 @@ class AP_MotorsHeli_RSC {
 
 private:
     uint64_t        _last_update_us;
+    uint8_t         _instance;
```

Comment:
```suggestion
    const uint8_t         _instance;
```
The other stuff set in the constructor should probably be const too....